src/wp-admin/includes/translation-install.php

```diff
@@ -35,23 +34,25 @@ function translations_api( $type, $args = null ) {
 	$res = apply_filters( 'translations_api', false, $type, $args );
 
 	if ( false === $res ) {
-		$url = $http_url = 'https://api.wordpress.org/translations/' . $type . '/1.0/';
-
+		$stats = array(
+			'wp_version' => $cp_version,
+			'locale'     => get_locale(),
+			'version'    => $args['version'], // Version of plugin, theme or core
+		);
 		$options = array(
 			'timeout' => 3,
-			'body' => array(
-				'wp_version' => $wp_version,
-				'locale'     => get_locale(),
-				'version'    => $args['version'], // Version of plugin, theme or core
-			),
 		);
-
-		if ( 'core' !== $type ) {
+		if( 'core' !== $type ){
```

Comment:
... and here, write `$stats['wp_version'] = $wp_version;` ...

Reply:
ok